standard/unsafe-code.md

```diff
@@ -967,6 +967,8 @@ In a member access of the form `E.I`, if `E` is of a struct type and a member lo
 
 The subsequent elements of the fixed-size buffer can be accessed using pointer operations from the first element. Unlike access to arrays, access to the elements of a fixed-size buffer is an unsafe operation and is not range checked.
 
+In an element access of the form `E[I]`, if `E` is a fixed-sized buffer, and `I` is implicitly convertible to `int`, then `E[I]` is evaluated exactly as the *pointer_element_access* (§22.6.4), `P[I]`, where `P` is the address of the fixed-sized buffer, `E`.
```

Comment:
I thought I'd added a review comment earlier, but apparently not. Do we need some of the same restrictions as in member access, but for element access, e.g. `E` not being a value, and occurring in an unsafe context?

Reply:
I don't think so. If `E` is a *fixed-size buffer*, it can't be a value. Also, fixed size buffers can only occur in unsafe code. 

I think it would be redundant.